gTea/app/src/main/java/com/nechaieva/gtea/OrderFragment.java

```diff
@@ -9,7 +9,7 @@
 import androidx.fragment.app.Fragment;
 import androidx.navigation.fragment.NavHostFragment;
 
-public class SecondFragment extends Fragment {
+public class OrderFragment extends Fragment {
```

Comment:
A more descriptive commit description could be "Renames NavigationFragment to Menu and SecondFragment to OrderFragment.

Reply:
Should I change it? That will probably require a force push after the operation.

Reply:
No, it's just FYI for subsequent commits.